Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DOCKER   ?= docker
+CONTAINER_TOOL ?= docker
```

Comment:
Do we want to add a note to the README on how to get started using podman?

Reply:
Yes, that's a good idea. I will update the PR.